tests/xtimer_drift/main.c

```diff
@@ -127,6 +131,11 @@ void *worker_thread(void *arg)
 
 int main(void)
 {
+    gpio_init(GPIO_PIN(PORT_F, 4), GPIO_OUT);
```

Comment:
It is not obvious to me where you are using this, and the following three ports.